plugins/modules/aci_interface_policy_leaf_policy_group.py

```diff
@@ -396,6 +481,21 @@ def main():
  (leaf access port policy group), if used\
  assign null to it (port_channel_policy: null)."
         )
+    invalid_parameters = {
+        "transceiver_policy": "transceiver_policy is not a valid parameter for link/node (Port Channel, Virtual Port Channel),\
+ if used assign null to it (transceiver_policy: null).",
+        "port_authentication": "port_authentication is not a valid parameter for link/node (Port Channel, Virtual Port Channel),\
+ if used assign null to it (port_authentication: null).",
+        "dwdm": "dwdm is not a valid parameter for link/node (Port Channel, Virtual Port Channel),\
+ if used assign null to it (dwdm: null).",
+        "poe_interface_policy": "poe_interface_policy is not a valid parameter for link/node (Port Channel, Virtual Port Channel),\
+ if used assign null to it (poe_interface_policy: null).",
+    }
+
+    if lag_type in ["link", "node"]:
+        for param, message in invalid_parameters.items():
+            if locals().get(param) is not None:
+                aci.fail_json(message)
```

Comment:
Why is lag_type set to None below when 'leaf' is assigned to it?

Reply:
lag_type is part of target_filter, so need to be reset here.